modules/nodes-pods-configuring-restart.adoc

```diff
@@ -43,6 +43,10 @@ restart, use a restart policy of `Never`.
 If an entire pod fails, {product-title} starts a new pod. Developers must address the possibility that applications might be restarted in a new pod. In particular,
 applications must handle temporary files, locks, incomplete output, and so forth caused by previous runs.
 
+[NOTE]
+====
+Kubernetes architecture expects reliable endpoints from cloud providers. When a cloud provider is down, the kubelet prevents {product-title} from restarting. If your underlying cloud provider is unavailable, it may be helpful to disable integration with {product-title} and configure the kubelet as a bare-metal or no-cloud environment.
```

Comment:
s/may/might
s/bare-metal/bare metal
Is it helpful, or will you be able to restart your cluster if you disable the integration?

To my knowledge, you can't change the configuration of a 4.x cluster to bare metal after you've provisioned it.

Reply:
@kalexand-rh I applied your first two edits to master-3 only [PR16068](https://github.com/openshift/openshift-docs/pull/16068/) while awaiting SME feedback for 4.x impact. 

Not sure the answer to your question ("Is it helpful"), so hope @derekwaynecarr can also confirm if this is accurate when he reviews.

Reply:
@kalexand-rh i see no value in taking an existing cluster that was configured to use a cloud provider and transform it to disable the cloud provider.  make the underlying cloud environment reliable, or install a new cluster.  it's probably possible to do that in 3.x, but I am certain it was never tested, and depending on how the cluster is used, it would cause chaos.  i think its more accurate to say that "If the underling cloud provider endpoints are not reliable, do not install a cluster using cloud provider integration.  Install the cluster as if it was in a no-cloud environment.  It is not recommended to toggle cloud provider integration on or off in an installed cluster."